functests/1_performance/performance.go

```diff
@@ -192,13 +195,55 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 		})
 
 		It("[test_id:35363][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] stalld daemon is running on the host", func() {
-			Skip("until bugs https://bugzilla.redhat.com/show_bug.cgi?id=1912118 and https://bugzilla.redhat.com/show_bug.cgi?id=1903302 fixed")
+			By("Checking minimal required cluster version")
+			clusterVersion := ""
+			cv := &v1.ClusterVersion{}
+			key := types.NamespacedName{
+				Name: "version",
+			}
+			err := testclient.Client.Get(context.TODO(), key, cv)
+			Expect(err).ToNot(HaveOccurred(), "cannot find the Cluster version object "+key.String())
+			clusterVersion = cv.Status.Desired.Version
+			currentClusterVersion, err := version.NewVersion(clusterVersion)
+			Expect(err).ToNot(HaveOccurred())
+			requiredStalldClusterVersion, err := version.NewVersion("4.7.6")
+			Expect(err).ToNot(HaveOccurred())
+			if currentClusterVersion.GreaterThanOrEqual(requiredStalldClusterVersion) {
+				for _, node := range workerRTNodes {
+					tuned := tunedForNode(&node)
+					_, err = pods.ExecCommandOnPod(tuned, []string{"pidof", "stalld"})
+					Expect(err).ToNot(HaveOccurred())
+				}
+			}
+
+			By("Checking minmal required kernel version")
+			stalldMinimumKernelVersionPrefix := "4.18.0-240.22"
```

Comment:
Does it still relevant to check the kernel version? I can not see that you check the kernel version under the controller.

Reply:
I am considering to leave that here since it already detected wrong versions in openshift releases. consider this as a double check . just need to see why its currently failing

Reply:
I would like to see a comment explaining the purpose then, like you just did here in the review.

Reply:
done